docs/contributing/miscellaneous.rst

```diff
@@ -7,6 +7,7 @@ Miscellaneous
 Core committers
 ------------------------
 
+.. _community_cla:
```

Comment:
Just a nitpick @abegong I called this ref `community_cla` because the top ref is `_community_miscellaneous`, but everything else uses `contributing_xxx` - might want to rename that for consistency.

Reply:
:+1:

will do